server/services/RunService.js

```diff
@@ -38,13 +38,281 @@ const db = require('../models/index');
  *
  */
 
+
 /* eslint-disable no-unused-vars */
 async function configureRuns({
     test_version_id,
     apg_example_ids,
     at_browser_pairs
 }) {
+    // if (!t) {
+    //     t = await db.sequelize.transaction();
```

Comment:
I don't know what is going on here. 

But I also think probably the whole thing should be wrapped in a transaction. Because we don't want to update half the things and fail. Or have a problematic race condition.